### DISCUSSION TITLE
Welcome to ath10k-ct Discussions!

### DISCUSSION THREAD

**Discussion by greearb** [greearb/ath10k-ct#175] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anjanpaul09** [comment]
Hello Everyone,
I am using the openwrt package aircrack-ng on DR4029(IPQ4029) hardware which uses ath10k-ct. I am hoping it should be using injection API only. I could not see the deauth packets going out as no capture in sniffer. Here is my output.
# aireplay-ng --deauth 0 -a 3C:84:6A:A7:92:D6 -c 64:5a:04:8c:9e:64 mon0 
21:08:43  Waiting for beacon frame (BSSID: 3C:84:6A:A7:92:D6) on channel 2
21:08:43  Sending 64 directed DeAuth (code 7). STMAC: [64:5A:04:8C:9E:64] [ 0| 0 ACKs]
21:08:45  Sending 64 directed DeAuth (code 7). STMAC: [64:5A:04:8C:9E:64] [ 5| 12 ACKs]
21:08:47  Sending 64 directed DeAuth (code 7). STMAC: [64:5A:04:8C:9E:64] [ 6| 14 ACKs]
21:08:49  Sending 64 directed DeAuth (code 7). STMAC: [64:5A:04:8C:9E:64] [ 8| 12 ACKs]
21:08:52  Sending 64 directed DeAuth (code 7). STMAC: [64:5A:04:8C:9E:64] [ 0| 0 ACKs]
21:08:54  Sending 64 directed DeAuth (code 7). STMAC: [64:5A:04:8C:9E:64] [ 0| 0 ACKs]
I tried enabling rawmode=1 but it says firmware support is not present.
The same command works fine with ath9k on QCA9565 which is there on my laptop.
Please suggest a way out. Thanks.

**Comment by Hodgy75** [comment]
Hello. I'm hoping this ath10k driver can solve my wifi problems on Dell XPS13 9370. I'm having trouble because I don't have the skills to build the driver manually in a terminal. So I run ./build_me.sh but I do not get the option to build for the latest kernel, presumably because this is not included in the build_me script. Can someone help me compile the driver for kernel 5.15? Thank you so much for any help.

Ian
